rep-0154.rst

```diff
@@ -0,0 +1,104 @@
+REP: 154
+Title: test_abi support for ROS distribution files
+Author: Jose Luis Rivero <jrivero@openrobotics.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 21-Nov-2018
+Post-History: 21-Nov-2019
+
+
+.. contents::
+
+Abstract
+========
+This REP updates the specification of the ROS distribution files facilitated in
+the building, packaging, testing and documenting process.
+
+The intention is to annotate a ROS distribution with more information,
+particularly with the option of running an ABI/API analysis for pull requests
+and devel jobs.
+
+This REP is an extension to the file format originally defined in REP 143[1]_
+and extended in REP 153[2]_. It currently does not repeat the content of REPs
+143 and 153 but only states the differences.
+
+
+Motivation
+==========
+
+Since the creation of ROS1 some developers and users have been looking into
```

Comment:
I would rephrase this more in the following direction:

* While maintainers aim to provide API / ABI stability within a ROS distribution there is no automated tool atm which checks that.
* Until now patches are manually reviewed and their ABI effect is judged by the reviewer.
* Providing an automated check will ensure that undesired ABI changes are catched.
* The feedback through PR as well as devel jobs allows each maintainer to be aware of the effect and make an informed decision.

I would't mention any policy and what is usally done since that isn't the actual motivation.

Reply:
Thanks for the pointers. I've rephrase the whole thing in a5877ef63a52d5ecbc83e8e82b99a225890b2180. Let me know if we need to change or modify anything more.